src/model/specific_path/Path.java

```diff
@@ -1,13 +1,67 @@
 package model.specific_path;
 
+import java.awt.Point;
+import java.util.ArrayList;
+
 import model.Shape;
 import model.ShapeName;
 
 public class Path extends Shape {
+	
```

Comment:
Path is in fact OpenPath, and it cannot be stuff such as circles for example. Maybe change the name to OpenPath?